tests/piqueserver/test_badmin.py

```diff
@@ -25,13 +27,17 @@ def test_score_grief2(self):
 
         self.assertEqual(0,badmin.score_grief(connection,player))
 
+    # Give grief score to player - blocks destroyed
     def test_score_grief3(self):
         connection = Mock()
         connection.colors = True
         connection.protocol.players = []
 
         player = Mock()
-        player.blocks_removed = [[110,None]]
```

Comment:
Please make sure your files follow the [pep8](https://www.python.org/dev/peps/pep-0008/) style in the future. You can use tools such as `flake8` to ensure they conform or even use `autopep8` to format it automatically.